src/frontend/core/scheduler/JobScheduler.cpp

```diff
@@ -107,6 +107,10 @@ void JobScheduler::processJob(JobRequest request, SQLiteDBInterface sqlite, Perf
 void JobScheduler::executeJob(JobRequest request, SQLiteDBInterface sqlite, PerformanceSQLiteDBInterface perfDB) {
     ExecutorFactory *executorFactory = new ExecutorFactory(sqlite, perfDB);
     AbstractExecutor *abstractExecutor = executorFactory->getExecutor(request);
+    if (abstractExecutor == nullptr) {
+        jobScheduler_Logger.error("abstractExecutor is null");
```

Comment:
Can we have more meaningful log error message here? When someone looks at the log file its bit confusing what this message means.